x-pack/plugins/ml/server/routes/system.ts

```diff
@@ -97,6 +97,12 @@ export function systemRoutes(
     .addVersion(
       {
         version: '1',
+        security: {
```

Comment:
this route should use the tag `access:ml:canGetMlInfo`

Reply:
This is called by other plugins and so will be safer to remain untagged and rely on the es privileges